app/screens/settings/notification_settings_auto_responder/notification_settings_auto_responder.js

```diff
@@ -65,6 +67,14 @@ export default class NotificationSettingsAutoResponder extends PureComponent {
         this.saveUserNotifyProps();
     }
 
+    componentDidMount() {
+        setTimeout(() => {
+            requestAnimationFrame(() => {
+                this.autoresponderRef.current.focus();
```

Comment:
```suggestion
                if (this.autoresponderRef?.current) {
                    this.autoresponderRef.current.focus();
                }
```

Reply:
0/5 on this, but if we're wanting to go this route, we could go further to `this.autoresponderRef?.current?.focus();`

Reply:
Actually, `this.autoresponderRef` will never be null since it's created in the constructor, so `this.autoresponderRef.current?.focus();` would be the correct way to do this

Reply:
Updated the if check to check for current before resolving the focus.